lib/service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -279,6 +279,23 @@ public Response resetMappings() {
 
         try {
             AtlasUtil.deleteDirectoryContents(mappingFolderPath.toFile());
+        } catch (Exception e) {
+            throw new WebApplicationException(e.getMessage(), e, Status.INTERNAL_SERVER_ERROR);
+        }
+        return Response.ok().build();
+    }
+
+    @DELETE
+    @Path("/fieldActions/RESET_LIBS")
```

Comment:
Can we use like `resetLibs` format to follow others?

Reply:
Do you mean put it under the path /mappings/RESET_LIBS instead of /fieldActions/RESET_LIBS?  The lib jars are for custom field actions more than mappings.  I can use /mappings path of you want though

Reply:
Ah wait that's also the one, because the jar could also contains Java Document class right?

I originally meant about the capital/small letters on `RESET_LIBS`, others are all using `capitalFirstCharWithStartingSmallLetter`, without underscore.